lib/sha1.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+var crypto = require('crypto');
+
+function sha1(bytes) {
+  if (Array.isArray(bytes)) bytes = Buffer.from(bytes);
```

Comment:
You'll want to explicitly create a buffer from a string, so you can specify the encoding (see PR #190).